node/core/provisioner/src/lib.rs

```diff
@@ -269,6 +269,15 @@ impl ProvisionerJob {
 			self.metrics.on_inherent_data_request(Err(()));
 		} else {
 			self.metrics.on_inherent_data_request(Ok(()));
+			gum::debug!(
+				target: LOG_TARGET,
+				signed_bitfield_count = self.signed_bitfields.len(),
+				signed_bitfields = ?self.signed_bitfields,
+				backed_candidates_count = self.backed_candidates.len(),
+				backed_candidates = ?self.backed_candidates,
```

Comment:
They are only printed once per block by the block producer, so that's a tradeoff imho is reasonable

@tdimitrov we should reduce the print of the backed candidates down to their hashes, I suggest reducing to trace-level and printing a reduced set in debug, aligned with what @eskimor suggested

Reply:
> They are only printed once per block by the block producer, so that's a tradeoff imho is reasonable

The provisioner is triggered by every node on every block for some reason. That wasn't the original behavior, but somewhere during an inherent-data refactoring in Substrate in 2021 it changed.